.github/actions/spell-check/expect.txt

```diff
@@ -1087,6 +1090,7 @@ IPrincipal
 IProgram
 IPublic
 IQuery
+IReadOnlyDictionary
```

Comment:
This should be `IRead` -- the spell checker parsed `IReadOnlyDictionary` as `IRead` + `Only` + `Dictionary` and was happy w/ 2/3 of them

Reply:
Thanks, yes I have notice this